stream_alert/rule_processor/firehose.py

```diff
@@ -155,11 +154,8 @@ def sanitize_keys(cls, record):
 
     def _backoff_handler_firehose_reset(self, details):
         """Custom backoff handler to re-instantiate the Firehose Client"""
-        LOGGER.info('[Backoff]: Calling \'%s\' again in %f seconds with %d tries so far',
-                    details['target'].__name__,
-                    details['wait'],
-                    details['tries'])
-        self._reset_firehose_client()
+        backoff_handler(debug_only=False)(details)
+        self._firehose_client = self._create_firehose_client()
```

Comment:
After some more thought, we may not need to do this on backoff. That was an attempt to fix the issue at hand, which didn't end up working.

Reply:
I was thinking the same but chose to leave it for.. happy to remove it if you think we should?

Reply:
We can remove it to boost the perf